packages/libdash/libdash.0.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "libdash"
+version: "0.1"
+synopsis: "Bindings to the dash shell's parser"
+maintainer: "Michael Greenberg <michael.greenberg@pomona.edu>"
+authors: "Michael Greenberg <michael.greenberg@pomona.edu>"
+license: "BSD"
+homepage: "https://github.com/mgree/libdash"
+bug-reports: "https://github.com/mgree/libdash/issues"
+depends: [
+  "ocaml" {>= "4.0.7"}
+  "ctypes" {= "0.11.5"}
```

Comment:
```suggestion
  "ctypes" {>= "0.11.5"}
```
Is there a reason you have a constraint so strict?

Reply:
I haven't been able to get my whole system to build with later ctypes... but this library should be fine.